eip-0001.md

```diff
@@ -30,19 +33,22 @@ A new request to scan is initiated which registers said scan to be checked for a
 A predicate (function which returns a boolean value for a box) is required to register a scan.
 Predicates available are:
 
-* `CONTAINS(register, bytes)` returns true if certain register contains given bytes. If *register* is missed, R1 will be scanned
-* `EQUALS(register, bytes)`returns true if certain register contains only given bytes. If *register* is missed, R1 will be scanned
+* `CONTAINS(register, value)` returns true if certain register contains given value. If *register* argument is missed, R1 (script register) will be scanned
+* `EQUALS(register, value)`returns true if certain register contains only given value. If *register* argument is missed, R1 will be scanned
 * `OFTYPE(register, typeId)` - if certain register is of certain type
 * `CONTAINS_ASSET(assetId)`- if a box contains asset with a given id
 * `AND(predicate1, predicate2, ..., predicateN)` - if all the children predicates are true
 * `OR(predicate1, predicate2, ..., predicateN)` - if one of the children predicates is true
 
+*value* field in the predicates above is about encoded sigma value, *assetId* is about just 32-bytes long byte array.
 
-The following is an example of a predicate which states that the registered scan will search for boxes that contain a given asset and contains the provided bytes in R1.
+The following is an example of a predicate which states that the registered scan will search for boxes that contain a 
+given asset and also the provided bytes in R1 (*"0e24..."* in the example is a byte array which contains a script 
+represented in ErgoTree form).
 ```haskell
 AND(
```

Comment:
Why haskell and not Scala?

Reply:
It is not Scala as well, but highlighting looks good